db/version_set.h

```diff
@@ -1032,10 +1033,11 @@ class VersionSet {
 
   // ApproximateSize helper
   uint64_t ApproximateSizeLevel0(Version* v, const LevelFilesBrief& files_brief,
-                                 const Slice& start, const Slice& end);
+                                 const Slice& start, const Slice& end,
+                                 bool for_compaction = false);
```

Comment:
Can we do without default value of arguments?

Reply:
I removed the default values.